juju/controller.py

```diff
@@ -193,7 +193,7 @@ async def disconnect(self):
         """Shut down the watcher task and close websockets.
 
         """
-        await self._connector.disconnect()
+        await self._connector.disconnect(entity='controller')
```

Comment:
Here you use 'controller', but later you use 'Model'. We should at least be consistent about the capitalization

Reply:
Oh yeah good catch, thanks